java/src/main/java/com/google/cloud/dataproc/templates/mongo/MongoToBQ.java

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
please relocate this to dataproc-templates/java/src/main/java/com/google/cloud/dataproc/templates
/databases/